specs/disciplina/educator/api/educator.yaml

```diff
@@ -695,7 +745,39 @@ paths:
           description: Request body is invalid
           schema:
             $ref: "#/definitions/ErrResponse"
+        401:
+          $ref: "#/responses/Unauthorized"
 
+securityDefinitions:
+  EducatorAuth:
+    type: apiKey
+    name: Authorization
+    in: header
+    description: |
+      Authentication system requires a signed JWT in Authorization header in `Bearer <JWT>` format. Example header:
+      ```
+      Authorization: Bearer eyJhbGciOiJFZERTQSJ9.eyJkYXQiOnsiYWRQYXRoIjoiL2FwaS9lZHVjYXRvci92MS9zdHVkZW50cyIsImFkVGltZSI6IjIwOTktMDgtMTBUMTM6MjY6NTkuNDYxOTk4MTM2WiJ9fQ.gU032BGkGwllzPtBLxsvXNZttI6qmEh4M2DGfdiIV6k2L_SuPNR8-YHGphQ2_bytOSKl4Hf8LKPt9g0wNjJbAQ
+      ```
+      The JWT should be signed by the educator's private key and contain one claim "dat". The claim should have an object inside with two fields "adPath" and "adTime'. "adPath" should contain the path to the endpoint as it is going to be in the raw request and "adTime" should have the current time in ISO-8601 format. Example JWT object:
+      ```
+      {
+        "dat": {
+          "adTime": "2018-08-10T13:22:40.461998136Z",
+          "adPath": "/api/educator/v1/students"
```

Comment:
Typo?

Reply:
Why? Isn't that a valid endpoint for Educator?

Reply:
Hm, okay